.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,43 @@
+name: "\U0001F41E Bug report"
+description: Report an issue or bug
+# Labels must be prepared in advance, otherwise no labels are applied
+labels: ["pending triage"]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to fill out this bug report!
```

Comment:
Nitro's template shows the content below instead:

> If bug-report is related to Nuxt framework, please open an issue directly there:
> https://github.com/nuxt/framework/issues/new/choose

My PR shows the same content as Nitro's feature request template.

Which should I set up?

Reply:
cf. https://github.com/unjs/template/pull/30#discussion_r1035285828